src/System.Private.Uri/src/System.Private.Uri.csproj

```diff
@@ -161,6 +161,9 @@
     <Compile Include="$(CommonPath)\Interop\Windows\kernel32\Interop.GetLastError.cs" Condition="'$(TargetsWindows)'=='true'">
       <Link>Common\Interop\Windows\kernel32\Interop.GetLastError.cs</Link>
     </Compile>
+    <Compile Include="$(CommonPath)\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs" Condition="'$(TargetsUnix)'=='true'">
+      <Link>Common\Microsoft\Win32\SafeHandles\SafeFileHandle.Unix.cs</Link>
```

Comment:
I tried building Unix uap101aot version of System.Private.Uri (referenced from S.R.Extensions) and that failed due to SafeFileHandle not defined.  Are you saying we'll never build uap101aot for Unix?